domain_test.go

```diff
@@ -38,21 +38,158 @@ func TestDomainCreate(t *testing.T) {
 	autopilot.Equals(t, "Z2lkOi8vb3BzbGV2ZWwvRW50aXR5T2JqZWN0LzMw", string(result.Id))
 }
 
+func TestDomainGetSystems(t *testing.T) {
+	// Arrange
+	requests := []TestRequest{
+		{`{"query": "query DomainChildSystemsList($after:String!$domain:IdentifierInput!$first:Int!){account{domain(input: $domain){childSystems(after: $after, first: $first){nodes{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}},parent{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}}}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}}}}",
```

Comment:
`totalCount` should no longer be in these queries

Reply:
Went around and removed total count but had to remove it from the tag connection too which will break other tests.  Made the required changes to get service's GetTags test to pass but had to sum it and now I'm wondering why we don't have to sum it on the domain side? 🤷

Reply:
It would have been nicer to have done this in a separate PR so its isolated, easier to review and easier to rollback.

If you have time and its not too much trouble extracting this a separate PR would be worth it.

I'm not sure what you mean by the other part you are wondering about - i'll have to dig in to see/understand.

Reply:
Just the refactor of all the other things that have totalcount that are not domains and systems.  I understand you concern with the fixtures of dependent types might still have total count and thats fine - we can refactor those later.

The problem is we could potentially merge the other refactor into main - but this domains stuff needs to be held out.

Also problematic is cluttering a PR with multiple changes - each PR should be small targeted changes - in hindsight it would be even better if each Create, Update, Get was each their own PR as the functionality should all be isolated and none of the code dependent upon each other.

Also in hindsight - We could stand to break our tickets down more like this so we make tickets to PRs more cleanly.

Reply:
Yeah, that's actually a good point.  Would have been nice to split up the ticket and could have been easier to divide the work that way.  I'll make sure to make the totalCount changes on the system side when I finish out that PR and I'll open a separate PR for the dependent types.